### ISSUE TITLE
[python3] build failure

### ISSUE THREAD

**Issue by ivanovmp** [microsoft/vcpkg#10735]
**Host Environment**
- OS: 64-bit Windows 10 (Russian language)
- Compiler: Microsoft ® C/C++ Optimizing Compiler Version 19.25.28612 for x86.

**To Reproduce**
`./vcpkg install python3:x86-windows`

**Failure logs** 
```
Computing installation plan...
The following packages will be built and installed:
    python3[core]:x86-windows
Starting package 1/1: python3:x86-windows
Building package python3[core]:x86-windows...
-- Using cached C:/misc/vcpkg/downloads/python-cpython-v3.7.3.tar.gz
-- Extracting source C:/misc/vcpkg/downloads/python-cpython-v3.7.3.tar.gz
-- Using source at C:/misc/vcpkg/buildtrees/python3/src/v3.7.3-1161ad7a91
-- Building C:/misc/vcpkg/buildtrees/python3/src/v3.7.3-1161ad7a91-Lib-Win/PCBuild/pythoncore.vcxproj for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: msbuild C:/misc/vcpkg/buildtrees/python3/src/v3.7.3-1161ad7a91-Lib-Win/PCBuild/pythoncore.vcxproj /p:Configuration=Release /t:Rebuild /p:Platform=Win32 /p:PlatformToolset=v142 /p:VCPkgLocalAppDataDisabled=true /p:UseIntelMKL=No /p:WindowsTargetPlatformVersion=10.0.18362.0 /m
    Working Directory: C:/misc/vcpkg/buildtrees/python3/x86-windows-rel
    Error code: 1
    See logs for more information:
      C:\misc\vcpkg\buildtrees\python3\build-x86-windows-rel-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_msbuild.cmake:119 (vcpkg_execute_required_process)
  ports/python3/portfile.cmake:46 (vcpkg_build_msbuild)
  scripts/ports.cmake:90 (include)


Error: Building package python3:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: python3:x86-windows
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.
```

Contents of `C:\misc\vcpkg\buildtrees\python3\build-x86-windows-rel-out.log`:
[build-x86-windows-rel-out.log](https://github.com/microsoft/vcpkg/files/4451837/build-x86-windows-rel-out.log)

**Additional context**
I find it quite suspicious that the log contains the following: `"git" is not recognized as an internal or external command operable program or batch file`. I had added `git` into `PATH`.

**Comment by Voskrese** [comment]
git тут не причем, ошибки при сборке модуля.
чтобы долго не искать причину, переустанови winsdk и msvc на максимум.
у меня таких ошибок не воникает.

и обязательно установите английский языковой пакет

**Comment by ivanovmp** [comment]
В Visual Studio 2019 и так установлены русский и английский языки, но я не знаю, как сделать так, чтобы при компиляции комментарии отображались на английском.

**Comment by ivanovmp** [comment]
I reinstalled winsdk (now it's version 10.0.18362.1) and Microsoft Visual C++, but the problem didn't vanish, still build failed because of a compilation error.

**Comment by LilyWangL** [comment]
@ivanovmp Thanks for posting this issue. Could you turn off "Beta: Use Unicode UTF-8.." and rebuild python3 again? You can get how to turn off it from here: https://stackoverflow.com/questions/56419639/what-does-beta-use-unicode-utf-8-for-worldwide-language-support-actually-do

**Comment by ivanovmp** [comment]
The checkbox was already unchecked.

![Безымянный](https://user-images.githubusercontent.com/28906405/79157098-a6076100-7ddc-11ea-93af-6385f01d22b4.png)

**Comment by Voskrese** [comment]
msbuild будет в логе на твоем писать, но это не страшно. vcpkg ставит vslang=1033 , только для этого нужен анг.язык пакет.
а ошибка у тебя по другой совсем причине.

**Comment by ivanovmp** [comment]
Ну и по какой?

**Comment by Voskrese** [comment]
```
         _math.c
         _pickle.c
     1>C:\misc\vcpkg\buildtrees\python3\src\v3.7.3-1161ad7a91-Lib-Win\Modules\_pickle.c(50,5): error C2059: syntax error: 'constant' [C:\misc\vcpkg\buildtrees\python3\src\v3.7.3-1161ad7a91-Lib-Win\PCBuild\pythoncore.vcxproj]
```

**Comment by ivanovmp** [comment]
Okay. I opened the project pythoncore.vcproj (that contained the error. I found out that there is a problem in `enum opcode`: it has a value UNICODE = 'V' which fails because UNICODE is already defined as 1. I tried to find the reasons, and it turned out that UNICODE was contained in the standard compilation line in my Visual Studio for some reason. I shall now try and remove it and see if python3[core] installs succesfully after that.
